packages/auth/src/config/app.ts

```diff
@@ -43,5 +43,7 @@ export const Config = {
   authServerDomain: envString('AUTH_SERVER_DOMAIN', 'http://localhost:3006'), // TODO: replace this with whatever frontend port ends up being
   waitTimeSeconds: envInt('WAIT_SECONDS', 5),
   cookieKey: envString('COOKIE_KEY', crypto.randomBytes(32).toString('hex')),
-  accessTokenExpirySeconds: envInt('ACCESS_TOKEN_EXPIRY_SECONDS', 10 * 60) // Default 10 minutes
+  accessTokenExpirySeconds: envInt('ACCESS_TOKEN_EXPIRY_SECONDS', 10 * 60), // Default 10 minutes
+  databaseCleanupWorkers: envInt('DATABASE_CLEANUP_WORKERS', 1),
+  accessTokenMinDaysBeforeDeletion: 30
```

Comment:
Should this be configurable?

Reply:
I thought 10 minutes was reasonable but it's not actually part of the spec. I can just hardcode it if you think that's cleaner, though.

Reply:
I'm not sure this default is actually necessary.
We currently only have `databaseCleanupRules` for `accessTokens`, for which `expiresIn` is [nullable](https://github.com/interledger/rafiki/blob/7e32e45bf9259837c95abb34fd110ac45712f2f3/packages/auth/migrations/20220504163005_create_access_token_table.js#L6) but we appear to always [define it](https://github.com/interledger/rafiki/blob/7e32e45bf9259837c95abb34fd110ac45712f2f3/packages/auth/src/accessToken/service.ts#L121).

I vote we make `expiresIn` non-nullable and remove `accessTokenMinDaysBeforeDeletion`

Reply:
I second the above ☝️

Reply:
I could remove `accessTokenMinDaysBeforeDeletion` and make `expiresIn` non-nullable, but it seems like the [spec](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-6.1) recommends against removing expired tokens immediately:

`Note that in many cases, the access token will have expired for
   regular use.  To facilitate token rotation, the AS SHOULD honor the
   rotation request of the expired access token since it is likely that
   the client instance is attempting to refresh the expired token.`

That makes me think that we'd still need to define a constant somewhere to prevent the auth service from cleaning up tokens before the client has a chance to request renewal of an expired token.

Maybe it doesn't need to be configurable, though. Let me know what you think

Reply:
Got it. Good point.
How about a configurable `accessTokenDeletionDays` that `processDatabaseCleanup` uses?
An access token would be deleted if (`createdAt` + `expiresIn` +`accessTokenDeletionDays`) < `now`.

Should the old access token always be deleted (expired or otherwise) by token rotation?

Reply:
Yeah, right now the old token is deleted during rotation (see #335 / #375)